### ISSUE TITLE
NullReferenceException when AudioSettings.OnAudioConfigurationChanged is invoked

### ISSUE THREAD

**Issue by RuudvanReenen** [RenderHeads/UnityPlugin-AVProVideo#1588]
### Please DO NOT LINK / ATTACH YOUR PROJECT FILES HERE

**Describe the issue**
When AudioSettings.OnAudioConfigurationChanged is invoked, this sometimes results in a NullReferenceException
Stacktrace:
RenderHeads.Media.AVProVideo.AudioOutput.OnAudioConfigurationChanged (System.Boolean deviceChanged) (at <b1e72ef849d84271b8e2ecbce4859854>:0)
UnityEngine.AudioSettings.InvokeOnAudioConfigurationChanged (System.Boolean deviceWasChanged) (at <7d2ecd17c9484b65bfc4e3e4501d5360>:0)

This is the method that's invoked:
		void OnAudioConfigurationChanged(bool deviceChanged)
		{
			if (_mediaPlayer.Control == null)
				return;
			_mediaPlayer.Control.AudioConfigurationChanged(deviceChanged);
		}
Apparently _mediaPlayer itself can be null, but this isn't checked.
Can I simply change this to: if (_mediaPlayer == null || _mediaPlayer.Control == null) or should _mediaPlayer never be null?

**Your Setup (please complete the following information):**
 - Unity version: 2021.3.18f1
 - AVPro Video version (number and edition (trial/core/ultra/enterprise)): 2.7.1 Ultra
 - Operating system version: Windows
 - Device model: 
 - Video specs (resolution, frame-rate, codec, file size): 

**To Reproduce**
1. I'm not sure how to reproduce this reliably, the error occurs only sometimes
2. 
3. 

**Logs**
If applicable, add error logs to help explain your problem.

**Screenshots**

**Videos**
If applicable, add a copy of your video or the URL

### Please DO NOT LINK / ATTACH YOUR PROJECT FILES HERE
Instead email the link to us unitysupport@renderheads.com

**Comment by Ste-RH** [comment]
Yes, you can safety add that check.  We will add it to the codebase here and it will be included in the next build.

Thanks for reporting this!
